### ISSUE TITLE
Compilation error while running test

### ISSUE THREAD

**Issue by slashmili** [rtfeldman/node-test-runner#163]
Hey,

I have some problem getting compilation error during running the tests. I would expect that elm-test runner provides me the errors so I can use the great error message from elm compiler

Environment: 
```
elm test --version
0.18.6
elm --version
0.18.0
node --version
v6.6.0
```

```
elm-test: 
(4.1.0)
```

So the problem that I have is when I run the test, gets an error like : 
```
elm test
Compilation failed while attempting to build /Users/milad/my_app/tests/UserDataTest.elm
```

If I run the command with `--verbose` option I get :  
```
elm test --verbose
Running elm-make /Users/milad/my_app/tests/UserDataTest.elm --yes
Compilation failed while attempting to build /Users/milad/my_app/tests/UserDataTest.elm
```

It doesn't print out the great elm compiler output but still tells me that I can run `elm-make` command and probably get the errors. However that doesn't work: 

```
elm-make /Users/milad/my_app/tests/UserDataTest.elm --output /dev/null --yes
I cannot find module 'Test'.

Module 'UserDataTest' is trying to import it.

Potential problems could be:
  * Misspelled the module name
  * Need to add a source directory or new dependency to elm-package.json
```

Is there anyway to print out the error message in the console that runs `elm test` command?

**Comment by zwilias** [comment]
Is there a `"elm-community/elm-test": "4.0.0 <= v < 5.0.0"'` line in the `tests/elm-package.json` file?

**Comment by slashmili** [comment]
@zwilias Yes, actually I can run the test when everything is alright, it's just when elm can not compile the code, I get no feedback on what did I do wrong

This is the code: https://github.com/carloslima/gitwerk/tree/elm-test/apps/gitwerk_web/assets

```
/home/m/gitwerk/apps/gitwerk_web/assets $ elm test
Compilation failed while attempting to build /home/m/gitwerk/apps/gitwerk_web/assets/tests/UserDataTest.elm
```

**Comment by zwilias** [comment]
Right, I see. The initial issue will probably be solved through #161 so once a new release happens. The issue of manually compiling and getting a weird error is simply a matter of running that `elm-make` command from _within_ the `tests` directory :)

**Comment by slashmili** [comment]
Yup! #161 patch prints the error in the console.

Thanks
